packages/theme-check-common/src/checks/hardcoded-routes/index.ts

```diff
@@ -0,0 +1,107 @@
+import { NodeTypes, TextNode } from '@shopify/liquid-html-parser';
+import { Severity, SourceCodeType, LiquidCheckDefinition } from '../../types';
+import { isAttr, isNodeOfType, isValuedHtmlAttribute, ValuedHtmlAttribute } from '../utils';
+
+const ROUTES: Record<string, string> = {
```

Comment:
These are all the routes from the [routes obj docs](https://shopify.dev/docs/api/liquid/objects/routes) except `/`. 

I chose not to warn on `/` to align with how I saw route urls used in a best practices Shopify theme repo and because I wasn’t sure if it would have the same benefits (translations etc.)

Reply:
I believe that it may still be worth it based off what I'm reading here - https://shopify.dev/docs/api/liquid/objects/routes#routes

You can see this on my store where the default language is set to french
`https://mammoth-matcha-redirect.shopitest.com` redirects me to `https://mammoth-matcha-redirect.shopitest.com/en` since my browser is set to English

Reply:
That makes sense, thanks! Fixed